### ISSUE TITLE
For binary classification show both classes in description

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#58]
Currently for binary classification only weights for one class are shown:

![pic](https://cloud.githubusercontent.com/assets/107893/19987898/924ea8b8-a247-11e6-8c56-c62ccbc7b3bd.png)

It'd be nice to show other class name as well, maybe in the bottom of the table.

**Comment by Svampen** [comment]
Any comments on this?

**Comment by kmike** [comment]
@Svampen nobody worked on this feature yet.
